Run7/step03/B_protocol.cfg

```diff
@@ -0,0 +1,132 @@
+[ACQUIRE]
+bias1
+bias2
+bias3
+bias4
+dark
+sflat
+lambda
+flat
+persistence
+
+[BIAS1]
+ACQTYPE=bias
+ANNOTATION=0 sec extra delay
+COUNT= 5
+EXTRADELAY = 0
+
+[BIAS2]
+ACQTYPE=bias
+ANNOTATION=15 sec extra delay
+COUNT= 5
+EXTRADELAY = 15.0
+
+[BIAS3]
+ACQTYPE=bias
+ANNOTATION=30 sec extra delay
+COUNT= 5
+EXTRADELAY = 30
+
+[BIAS4]
+ACQTYPE=bias
+ANNOTATION=60 sec extra delay
+COUNT= 5
+EXTRADELAY = 60
+
+[DARK]
+SHUTTER = OPEN  # Shutter opens at start of acqusition, closes at end
```

Comment:
In [fp.py line 84](https://github.com/lsst-camera-dh/fp-scripts/blob/b07be5358a0afadd713b1f13b063914f897973ac/lib/fp.py#L84) it looks like the shutter mode is ignored for dark acquisitions, but might be less confusing to remove the SHUTTER = OPEN line here and for other dark acquisitions